packages/styles/radio-card.css

```diff
@@ -88,3 +88,7 @@
   display: flex;
   justify-content: center;
 }
+
+.cauldron--theme-dark .RadioCardGroup__Icon.Icon svg {
+  color: var(--accent-success);
```

Comment:
I tested the contrast at https://webaim.org/resources/contrastchecker/. I'm not sure if there's an easier way.
<img width="743" alt="Screen Shot 2022-09-16 at 4 21 30 PM" src="https://user-images.githubusercontent.com/3081483/190819454-128c8739-56ca-4258-a50a-2b2c4e85a667.png">